tests/src/ExpertExtender/ExpertExtender.Listrotator.tests.js

```diff
@@ -14,17 +14,11 @@
 		} );
 	}
 
-	testExpertExtenderExtension.constructor(
-		ExpertExtender.Listrotator,
-		new ExpertExtender.Listrotator( '', [ { value: 'value', label: 'label' } ] )
-	);
-	testExpertExtenderExtension.destroy(
-		ExpertExtender.Listrotator,
-		new ExpertExtender.Listrotator( '', [ { value: 'value', label: 'label' } ] )
-	);
-	testExpertExtenderExtension.init(
-		new ExpertExtender.Listrotator( '', [ { value: 'value', label: 'label' } ] )
-	);
+	function getInstance() {
+		return new ExpertExtender.Listrotator( '', [ { value: 'value', label: 'label' } ] );
+	}
+
+	testExpertExtenderExtension.all( ExpertExtender.Listrotator, getInstance );
```

Comment:
Why do you use a named function here, but an inlined closure in all the other cases above?